djangocms_versioning/cms_toolbars.py

```diff
@@ -299,7 +299,7 @@ def get_page_content(self, language=None):
         return get_latest_admin_viewable_content(self.page, language=language)
 
     def populate(self):
-        self.page = self.request.current_page or getattr(self.toolbar.obj, "page", None)
+        self.page = self.request.current_page
```

Comment:
What was the scenario where the page wasn't on the request and needed to be retrieved from the toolbar?

Reply:
I don't think there is a scenario. I changed it to ensure the identical behaviour as the core (we had several instances where versioning replicated and deviated, leading to problems). This change, however, is not integral to this PR, so I can undo it.
I want to avoid that the page menu appears if a user is at the endpoint of a custom model which has a `page` property.
@marksweb  What's your take: Change or keep?

Reply:
@fsbraun Keep it in in that case 👍

Reply:
Have reintroduced it!